code/src/components/TodoForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import uniqid from 'uniqid'
+
+import todos from '../reducers/todos'
+
+const TodoForm = () => {
+    const [value, setValue] = useState('')
+
+    const dispatch = useDispatch()
+
+    const onFormSubmit = (event) => {
+        event.preventDefault()
+
+        const newTodo = {
+            id: uniqid(),
+            description: value, 
+            isComplete: false
+        }
+
+        dispatch(todos.actions.addTodo(newTodo))
+        setValue('')
+    }
+
+    return (
+        <form className="add-task-form" onSubmit={onFormSubmit}>
+            <section className="add-task-container">
+                <button className="add-task-button" type="submit">+</button>
+                <input
```

Comment:
IMHO it would look better if you make this input stretch and fill all the line 😸